plugin/sdk/logger.py

```
@@ -0,0 +1,407 @@
+        log_level: int = DEFAULT_LOG_LEVEL,
+        max_bytes: int = DEFAULT_MAX_BYTES,
+        backup_count: int = DEFAULT_BACKUP_COUNT,
+        max_files: int = DEFAULT_MAX_FILES,
+        log_format: Optional[str] = None,
+    ):
+        """
+        初始化插件文件日志管理器
+        
+        Args:
+            plugin_id: 插件ID
+            plugin_dir: 插件目录路径（通常是plugin.toml所在目录）
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)